### PR TITLE
feat: Set capella at epoch=0

### PR DESCRIPTION
**Pull request by barnabasbusa**
Tracking https://github.com/ethpandaops/ethereum-genesis-generator/pull/46

Once this is merged, and we make a rc release, we can merge this PR too.

### CODE REVIEW

**Comment by barnabasbusa** [comment]
We now have a rc.1 version for the genesis generator. 
This is safe to be merged.

File: package_io/input_parser.star [hunk 1]
```diff
@@ -162,8 +158,7 @@ def default_network_params():
 		"deposit_contract_address":              "0x4242424242424242424242424242424242424242",
 		"seconds_per_slot":                      12,
 		"slots_per_epoch":                       32,
-		"genesis_delay":                         10,
-		"capella_fork_epoch":                    2,
+		"genesis_delay":                         120,
```
Comment (h4ck3rk3y) [inline package_io/input_parser.star:161]: can you set this to 10?

**Review by h4ck3rk3y — APPROVED** [review]
Approving assuming @parithosh  is okay with this new default.

File: package_io/input_parser.star (hunk 1 above)
Comment (barnabasbusa) [reply package_io/input_parser.star:161]: The reason I set it to 120, is to enable CL peering before genesis, so we have a better chance to have healthy peers when genesis hits already.

Comment (h4ck3rk3y) [reply package_io/input_parser.star:161]: Alrighty; you know best :)

**Pull request merged**